atspi-common/src/state.rs

```diff
@@ -219,6 +221,122 @@ pub enum State {
 	ReadOnly,
 }
 
+impl From<State> for String {
```

Comment:
note you added this conversion to be able to print it elsewhere, then maybe save a conversion and `impl Display`, but you perhaps have other uses in mind as well..

Reply:
Technically it is still for conversion. Basically, I just want to infallibly convert to a String for a conversion from a `Message` to a `T: GenericEvent`. I could unwrap a `serde` call here, but just not sure how I feel about that.

Reply:
Maybe don't do that. There are still `Message`s which we do not parse to an event. The two DBus messages with "s" signature. (forgot their names)
And if interfaces change, or there are signals we somehow don't yet know of, we do not want to panic unless we absolutely must. We want ATs to handle an error and carry on.